src/vstart.sh

```diff
@@ -232,6 +234,8 @@ usage=$usage"\t--bluestore-devs: comma-separated list of blockdevs to use for bl
 usage=$usage"\t--bluestore-zoned: blockdevs listed by --bluestore-devs are zoned devices (HM-SMR HDD or ZNS SSD)\n"
 usage=$usage"\t--inc-osd: append some more osds into existing vcluster\n"
 usage=$usage"\t--cephadm: enable cephadm orchestrator with ~/.ssh/id_rsa[.pub]\n"
+usage=$usage"\t--cephadm-hosts: set a list of hosts that should be assigned to the cephadm orchestrator\n"
```

Comment:
hm. we could generalize this a bit. if we use `--apply -i <spec.yaml>`, this would provide a hook to also deploy services, not just adding hosts. wdyt?

Reply:
mh, that would save us adding options in the long run I guess :+1: